diff --git a/src/components/BaseHead.astro b/src/components/BaseHead.astro
index f5ce3e6..d21026d 100644
--- a/src/components/BaseHead.astro
+++ b/src/components/BaseHead.astro
@@ -68,6 +68,9 @@ const fullTitle = title === siteName ? title : `${title} | ${siteName}`;
 <!-- RSS -->
 <link rel="alternate" type="application/rss+xml" title={siteName} href="/rss.xml" />
 
+<!-- Google AdSense -->
+<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-3372468923504472" crossorigin="anonymous"></script>
+
 <!-- Google tag (gtag.js) -->
 <script async src="https://www.googletagmanager.com/gtag/js?id=G-C71ZM54X4T"></script>
 <script>
